openpype/hosts/nuke/api/lib.py

```diff
@@ -1300,13 +1301,8 @@ def create_write_node(
 
     # build file path to workfiles
     fdir = str(anatomy_filled["work"]["folder"]).replace("\\", "/")
```

Comment:
is the `.replace("\\", "/")` actually needed?

Reply:
Yeah `.replace("\\", "/")` is actually needed since Nuke is supporting only forward slashes internally.

Reply:
Oh yeah I was more questioning why it would have been "\" at first place (we use Linux so we never have "\") but that makes sense, thank you!